src/test/java/org/jboss/xavier/integrations/route/MainRouteBuilder_S3Test.java

```diff
@@ -0,0 +1,68 @@
+package org.jboss.xavier.integrations.route;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.aws.s3.S3Constants;
+import org.apache.camel.test.spring.CamelSpringBootRunner;
+import org.apache.camel.test.spring.UseAdviceWith;
+import org.jboss.xavier.Application;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+
+import javax.inject.Inject;
+import java.util.UUID;
+
+@RunWith(CamelSpringBootRunner.class)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@UseAdviceWith // Disables automatic start of Camel context
+@SpringBootTest(classes = {Application.class})
+@ActiveProfiles("test")
+public class MainRouteBuilder_S3Test {
+    @Inject
+    CamelContext camelContext;
+
+    @Test
+    @Ignore
```

Comment:
Could this test (and hence this class) be removed?

Reply:
no, it's a good example on how to test aws-s3, and if for any reason we want to test again with any other things ( encription ) it's only a matter of changing it with credentials.
but if you want me to delete I'll do

Reply:
ok